### ISSUE TITLE
#cloud #vm #security

### ISSUE THREAD

**Issue by MYY-my** [Superalgos/Superalgos#1541]
There is a video on YouTube for installing superalgos on linux ubuntu in the google cloud platform. It would be nice if another readme file appeared in github, for installation on a virtual instance, with security settings for the instance.

**Comment by Luis-Fernando-Molina** [comment]
Good idea, just too much work at the moment. Would you do it? if so I would happily merge that PR.

**Comment by khultman** [comment]
Hi - Picking this up:

I've started building an infrastructure as code project to deploy Superalgos securely to aws, see: https://github.com/khultman/Superalgos-IaC

I'll be working on this over the next several weeks and getting it to a working deployable state.
